Master/healthcheck.js

```diff
@@ -0,0 +1,111 @@
+/* ----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ * 
+ * Various Event hub collector health checks.
+ * The last error code is EHUB000006
+ * 
+ * @end
+ * ----------------------------------------------------------------------------
+ */
+ 
+const async = require('async');
+const azure = require('azure');
+const parse = require('parse-key-value');
+
+function initArmEhub(master) {
+    const azureCreds = master.getApplicationTokenCredentials();
+    const subscriptionId = master.getConfigAttrs().subscription_id;
+    return azure.createEventHubManagementClient(azureCreds, subscriptionId);
+}
+
+function formatSdkError(master, alErrorCode, err) {
+    if (typeof err === 'string' || err instanceof String) {
+        return master.errorStatusFmt(alErrorCode, err);
+    } else if (typeof err === 'object') {
```

Comment:
Perhaps change this to checking instanceof Object?

Reply:
this check make a little sense because
```
var f = function() {return null;};
console.log('@@@@@', typeof f);    // Outputs: function
console.log('@@@@@', f instanceof Object); // Outputs: true 
```